common/src/chain/block/block_v1.rs

```diff
@@ -5,7 +5,7 @@ use crate::primitives::Idable;
 use crate::primitives::H256;
 use parity_scale_codec::Encode;
 use parity_scale_codec_derive::{Decode as DecodeDer, Encode as EncodeDer};
-#[derive(Debug, Clone, PartialEq, Eq, EncodeDer, DecodeDer)]
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, EncodeDer, DecodeDer)]
```

Comment:
I'd like to use `pub type BlockMap = BTreeMap<Id<Block>, BlockIndex>;` instead of reading directly from the block storage but then removed it because it improves complexity. Ord\PartialOrd was needed for BlockMap. 

Removed back PartialOrd and Ord also BlockMap

Reply:
It's needed for keys, not values.